src/app/shared/helpers/config.ts

```diff
@@ -14,7 +18,10 @@ export interface Environment {
   defaultSkillId: string;
 
   languages?: { tag: string, path: string }[];
-}
+
+  // The auth type used with API is either `useToken: true` or `useCookies: true` (not both).
```

Comment:
Templates are not able to work with discriminated unions, that's why this "trick" is used at several places (mainly in states) in the project (it took me a while to find a type-safe (and valid, and clean) way to work in template). 
But here for the config, it is not really used in template. It was more a way to have `if (appConfig.useTokens) ...` in code instead of a "if (appConfig.authType === 'token') ..."... and I find the first option more readable. But that's true that should probably be a discriminated union instead here.

Reply:
Done. 
But anyway,what was implemented was already discriminated types, but just using two booleans instead of one string. It was not valid (TS not compile) to defined both var true or both var false, or not giving the cookie parameters with "useCookies"..